microservices/password-manager/utils/charset-utils.ts

```diff
@@ -32,13 +34,14 @@ const mapNumberToPaddedBinaryString = (num: number) => {
     return num.toString(2).padStart(8, "0");
 }
 
-const encryptWithCharset = (charset: string, password: string): string => {
+const encryptWithCharset = (charset: string[], password: string): CommandOutput => {
```

Comment:
Why does this need to return a command output anyway? There is no command execution, everything is done in JS.